docker-compose/obkm/README.md

```diff
@@ -0,0 +1,52 @@
+# WSO2 Open Banking Open Banking Key Manager
```

Comment:
@Akila94 shall we include the fact that this pattern is for the SG specification, in the title name? WDYT?

Reply:
Added. Thanks.